src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -227,6 +233,9 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 	private _waitForSupportedExecutions: Promise<void>;
 	private _onDidRegisterSupportedExecutions: Emitter<void> = new Emitter();
 
+	// {{SQL CARBON EDIT}}
+	private lastRunTasksViewTask: TaskInfo;
```

Comment:
this name is kind of confusinng, do you mean recentTask? :)

Reply:
It's actually not the last or recent task, but the last one to show up on the tasks view, so I wanted to make that clear